docs/README.md

```diff
@@ -60,19 +60,17 @@ Read more in the [gRPC clients documentation](@clients/grpc/README.md).
 
 EventStoreDB offers a low-level protocol in the form of an asynchronous TCP protocol that exchanges protobuf objects. At present this protocol has adapters for .NET and the JVM.
 
-::: warning
-We plan to phase out the TCP protocol in later versions. Please consider migrating your applications that use the TCP protocol and refactor them to use gRPC instead.
+::: warning Deprecation Note
+TCP protocol will be available only through version 23.10. Please plan to refactor your applications that use the TCP protocol to use gRPC instead.
```

Comment:
```suggestion
TCP protocol will be available only through version 23.10. Please plan to migrate your applications that use the TCP client SDK to use the gRPC SDK instead.
```
it's more a migration than a refactor , 
using SDK in line with paragraph above